Auth0/CredentialsManager.swift

```diff
@@ -123,10 +123,9 @@ public struct CredentialsManager {
         guard
             let data = self.storage.data(forKey: self.storeKey),
             let credentials = NSKeyedUnarchiver.unarchiveObject(with: data) as? Credentials,
-            credentials.accessToken != nil,
-            let expiresIn = credentials.expiresIn
+            credentials.accessToken != nil
             else { return false }
-        return expiresIn > Date() || credentials.refreshToken != nil
+        return !self.hasExpired(credentials) || credentials.refreshToken != nil
```

Comment:
if neither AT or IDT have expired OR we have a refresh token as we can use that to gain new tokens so technically we have valid credentials. Although the RT could be invalid, not possible to know until calling renew.

Reply:
Why would we be using a refresh token for a new ID token? Particularly in the case of a native app.

Reply:
@joshcanhelp To obtain a fresh copy of the user profile, I imagine. Either that or use the access token to call /userinfo and obtain it that way.